libraries/DataFlash/DataFlash_Backend.cpp

```diff
@@ -260,6 +260,31 @@ bool DataFlash_Backend::Log_Write(const uint8_t msg_type, va_list arg_list, bool
     return WritePrioritisedBlock(buffer, msg_len, is_critical);
 }
 
+bool DataFlash_Backend::StartNewLogOK() const
+{
+    if (logging_started()) {
+        return false;
+    }
+    if (_front._log_bitmask == 0) {
+        return false;
+    }
+    if (_front.in_log_download()) {
+        return false;
+    }
+    if (!DataFlash_Backend::WritesOK()) {
```

Comment:
Any reason for this to be forcing the DataFlash_Backend WritesOK method instead of a possible overriding method?